packages/cli/src/commands/migrations/run/streams/update-stream.ts

```diff
@@ -89,9 +89,8 @@ export class UpdateStream extends Writable {
         payload.publish = 1;
       }
 
-      const updatedStory = await updateStory(this.options.space, storyId, payload);
-
-      if (updatedStory) {
+      const storyUpdated = this.options.dryRun || Boolean(await updateStory(this.options.space, storyId, payload));
```

Comment:
@maoberlehner what do you think about separating the condition and the updateStory response? Mostly for readability and in case the updated story needs to be used. Like:

```js
const updatedStory = !this.options.dryRun && await updateStory(this.options.space, storyId, payload);

const isStoryUpdated = Boolean(updatedStory);
```